kubemarine/core/flow.py

```diff
@@ -184,9 +184,8 @@ def run_tasks(resources: res.DynamicResources, tasks, cumulative_points=None, ta
     cluster = resources.cluster()
 
     if args.get('without_act', False):
+        cluster.context["dry_run"] = True
```

Comment:
I suggest to not recreate the inventory on the deployer. For example, if you run upgrade, the input cluster.yaml will be changed. This does not seem as "dry-run". I suggest to create separate file in dump/ and reflect the changes there.